src/test/java/org/takes/misc/VerboseIteratorTest.java

```diff
@@ -0,0 +1,72 @@
+package org.takes.misc;
+
+import static org.hamcrest.Matchers.equalTo;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.collection.IsIterableWithSize;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+/**
+ * 
```

Comment:
@m4solutions same here.
`Tests for {@link VerboseIterator}.` followed by 1 blank comment line.